README.md

```diff
@@ -1,2 +1,60 @@
-# To-Do-List
-This feature is a To-Do-list which enables us to add, clear and edit our list of tasks to be done
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> To-Do App
+
+## About Project
+
+In this feature, I built a to do list app which helps to list the tasks to be done at a particular time. One can check the done ones and deleted them too.
+
+## Built With
+
+- Major languages-HTML&CSS/JS
+- Frameworks-N/A
+- Technologies used- VSCode, Git, Github.
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+- You can download the file directly by downloading as a zip file directly from github.
+- Alternatively, you can clone the repository by running the git clone command.
+
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [ ] [OPTIONAL] Please, consider removing the unused section or fulfill them accordingly 👍